test/workload-deployment.sh

```diff
@@ -0,0 +1,140 @@
+kflex create imbs1 --type vcluster -p ocm
+
+echo "Create a Workload Description Space wds1 directly in KubeFlex. Similarly to before, -p kslight runs a post-create hook on the k8s control plane that starts an instance of a KSLight controller manager which connects to the wds1 front-end and the imbs1 OCM control plane back-end."
+echo "-------------------------------------------------------------------------"
+kflex create wds1 -p kslight
+
+echo "Register clusters with OCM"
+echo "-------------------------------------------------------------------------"
+./register_cluster1_cluster2_with_ocm.sh
+
+echo "Check all deployments and statefulsets running in the hosting cluster"
+echo "-------------------------------------------------------------------------"
+kubectl --context kind-kubeflex get deployments,statefulsets --all-namespaces
+
+echo "Check for available clusters with label location-group=edge"
+echo "-------------------------------------------------------------------------"
+kubectl --context imbs1 get managedclusters -l location-group=edge
+
+echo "Create a placement to deliver an app to all clusters in wds1."
+echo "This placement configuration determines where to deploy the workload by using the label selector expressions found in clusterSelectors. It also specifies what to deploy through the downsync.labelSelectors expressions. When there are multiple matchLabels expressions, they are combined using a logical AND operation. Conversely, when there are multiple labelSelectors, they are combined using a logical OR operation."
+echo "-------------------------------------------------------------------------"
+kubectl --context wds1 apply -f - <<EOF
+apiVersion: edge.kubestellar.io/v1alpha1
+kind: Placement
+metadata:
+  name: nginx-placement
+spec:
+  clusterSelectors:
+  - matchLabels: {"location-group":"edge"}
+  downsync:
+  - labelSelectors:
+    - matchLabels: {"app.kubernetes.io/name":"nginx"}
+EOF
+
+echo "Deploy the app"
+echo "-------------------------------------------------------------------------"
+kubectl --context wds1 apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/name: nginx
+  name: nginx
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  namespace: nginx
+  labels:
+    app.kubernetes.io/name: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: public.ecr.aws/nginx/nginx:latest 
+        ports:
+        - containerPort: 80
+EOF
+
+echo "Verify that manifestworks wrapping the objects have been created in the mailbox namespaces"
+echo "-------------------------------------------------------------------------"
+kubectl --context imbs1 get manifestworks -n cluster1
+kubectl --context imbs1 get manifestworks -n cluster2
```

Comment:
I see no checking here.

Reply:
Good point.  I added a check for the output.